crates/ibc/README.md

```diff
@@ -13,6 +13,12 @@ Implementation of the Inter-Blockchain Communication Protocol ([IBC]) in Rust.
 
 See documentation on [docs.rs][docs-link].
 
+## Third-party apps
```

Comment:
Doesn't the term "apps" seem a bit confusing? by implying it's only about IBC applications, when it actually includes both apps and clients.

Reply:
I was thinking we'd have another list for clients

Reply:
So, the Solo machine should go under the clients' list?

Reply:
oh boy, I meant client 😅 [2de6c97](https://github.com/cosmos/ibc-rs/pull/710/commits/2de6c97ccd9d3893807ed7d4a451e1d2ce44dae2)